### ISSUE TITLE
The example’s effect cannot be reproduced

### ISSUE THREAD

**Issue by xmcxcoc** [JarrentWu1031/AnimateAnyMesh#22]
python test_drive.py --data_dir ./examples --vae_dir ./checkpoints --rf_model_dir ./checkpoints --json_dir ./checkpoints/dvae_factors --rf_exp rf_model --rf_epoch f --seed 666 --test_name dragon --prompt "The object is flying" --export_format fbx --guidance_scale 5.0
WARN (translation.language): source/blender/blentranslation/intern/blt_lang.cc:214 BLT_lang_init: Falling back to standard locale ("C")
sh: warning: setlocale: LC_ALL: cannot change locale (en_US.UTF-8)
Loading unified training configuration...
Configuration loaded successfully.
Loading DyMeshVAE...
Loading checkpoint from: ./checkpoints/dvae/dvae_f.pth
Model weights loaded successfully.
DyMeshVAE loaded!!!
Loading RF Model...
Loading checkpoint from: ./checkpoints/rf_model/rf_epoch_f.pth
Model weights loaded successfully.
RF Model loaded!!!
19:57:52 | INFO: Data are loaded, start creating Blender stuff
19:57:52 | INFO: Blender create Mesh node Object_0
19:57:52 | INFO: glTF import finished in 0.03s
Loaded 1 mesh objects
Mesh: Object_2, Vertices: 9639
/root/zhicheng.chen/AnimateAnyMesh-main/test_drive.py:97: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.detach().clone() or sourceTensor.detach().clone().requires_grad_(True), rather than torch.tensor(sourceTensor).
  vertices, faces = torch.tensor(merged_verts, dtype=torch.float32), torch.tensor(merged_faces, dtype=torch.int64)
The number of sampled trajs is not specified, set to 1020 by default (8160 vertices in total)!!!
Animation prompt:  The object is flying
/root/miniconda3/envs/animateanymesh/lib/python3.11/contextlib.py:105: FutureWarning: `torch.backends.cuda.sdp_kernel()` is deprecated. In the future, this context manager will be removed. Please see `torch.nn.attention.sdpa_kernel()` for the new context manager, with updated signature.
  self.gen = func(*args, **kwds)
Start RF sampling...
RF sampling finished!!!
Setting up animations for all meshes...
Setting up scene...
Exporting animated mesh to FBX format...

FBX export starting... './output_videos/rf_model/dragon.fbx'
export finished in 0.0906 sec.
Successfully exported animated mesh to: ./output_videos/rf_model/dragon.fbx
Setting up rendering...
Rendering frames...
Fra:0 Mem:18.58M (Peak 18.58M) | Time:00:00.00 | Mem:0.00M, Peak:0.00M | Scene, ViewLayer | Synchronizing object | Object_2
Fra:0 Mem:18.58M (Peak 18.58M) | Time:00:00.00 | Mem:0.00M, Peak:0.00M | Scene, ViewLayer | Synchronizing object | Sun.002
Fra:0 Mem:18.59M (Peak 18.59M) | Time:00:00.00 | Mem:0.00M, Peak:0.00M | Scene, ViewLayer | Synchronizing object | Sun.008
...
Time: 00:00.82 (Saving: 00:00.02)

https://github.com/user-attachments/assets/747c5dd8-05de-43cb-8456-d942d12aab9a
